packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -165,29 +229,30 @@
   overflow-auto
   h-full
   focus-base;
-  padding: var(--calcite-panel-content-space, 0);
+  padding: var(--calcite-panel-space, var(--calcite-panel-content-space, 0));
```

Comment:
I don't think `--calcite-panel-space` is helpful here. Lets just style .content-top and .content-bottom with `--calcite-panel-content-space`

Reply:
give me a specific use case?

Reply:
I want to put a progress bar in the content-top area and not have space around it. I want to put stepper items in the content-top area and not have space around them. I want to place a full-width banner, aggregate messaging, or notice in the content-top area and not have space around it. There are dozens of use cases. If you'd like, I can remove these changes from this PR, and address the request from here down the road: https://github.com/Esri/calcite-design-system/issues/10380